src/templates/docs.js

```diff
@@ -130,15 +123,15 @@ export default class MDXRuntimeTest extends React.Component {
           {(config.features.showMetadata === true && mdx.frontmatter.showMetadata !== false) ||
             mdx.frontmatter.showMetadata === true ? (
             <div css={{ display: 'block' }}>
-              {mdx.parent.relativePath && docsRepo && docsLocationType && contentRootPath &&
-                <Authors 
+              {mdx.parent.relativePath && docsRepo && docsLocationType && contentRootPath && 
+                <PageMetadata 
                   path={mdx.parent.relativePath}
                   repo={docsRepo}
                   repoType={docsLocationType}
                   contentRootPath={contentRootPath}
                   locationPathname={this.props.location.pathname}
+                  timeToRead={mdx.timeToRead * 2} 
                   />}
-              <ReadingTime time={mdx.timeToRead * 2} />
             </div>
           ) : (
```

Comment:
I now this is the old code but can we remove `''`? when `mdx.parent.relativePath && docsRepo && docsLocationType && contentRootPath` return false?

Reply:
yes, we can